services/QuillLMS/client/app/bundles/Staff/helpers/evidence/routingHelpers.ts

```diff
@@ -72,6 +72,8 @@ export const getActivityHealthUrl = ({ activityId }) => {
 
 export const aggregatedActivityHealthsUrl = `evidence/activity_healths.json`;
 
+export const aggregatedPromptHealthsUrl = `evidence/prompt_healths.json`;
```

Comment:
Unless there is some expected conditional logic that will be applied to these, I'm not really sure it's worth pulling these out into exported variables